readme.md

```diff
@@ -1282,6 +1259,34 @@ module.exports = withTypescript(withSass({
 }))
 ```
 
+In order to extend our usage of `webpack`, you can define a function that extends its config via `next.config.js`.
+
+```js
+// next.config.js is not transformed by Babel. So you can only use javascript features supported by your version of Node.js.
+
+module.exports = {
+  webpack: (config, { buildId, dev, isServer, defaultLoaders }) => {
+    // Perform customizations to webpack config
+    // Important: return the modified config
+    return config
+  },
+  webpackDevMiddleware: config => {
+    // Perform customizations to webpack dev middleware config
+    // Important: return the modified config
+    return config
+  }
+}
+```
+
+The second argument to `webpack` is an object containing properties useful when customing the WebPack configuration:
+
+- `buildId` - `String` the build id used as a unique identifier between builds
+- `dev` - `Boolean` shows if the compilation is done in development mode
+- `isServer` - `Boolean` shows if the resulting configuration will be used for server side (`true`), or client size compilation (`false`).
+- `defaultLoaders` - `Object` Holds loaders Next.js uses internally, so that you can use them in custom configuration
```

Comment:
I'm confused. Does `defaultLoaders` hold the loader objects themselves, or just the configuration objects for the loaders?